code/packaging/app-config/src/main/resources/META-INF/cattle/process/machine-defaults.properties

```diff
@@ -1,6 +1,8 @@
 machine.service.executable=go-machine-service
 machine.execute=false
 
+machine.driver.config=file:///var/lib/cattle/machinedrivers.json
```

Comment:
Change to `file://var/lib/rancher/etc/machinedrivers.json`  @cloudnautique Does that path sound good?  Does it align with the config you did for SSL?